src/Http/Controllers/ResetController.php

```diff
@@ -2,21 +2,35 @@
 
 namespace Whitecube\LaravelCookieConsent\Http\Controllers;
 
-use Whitecube\LaravelCookieConsent\CookiesManager;
+use Illuminate\Support\Facades\Cookie;
 use Illuminate\Http\Request;
+use Whitecube\LaravelCookieConsent\CookiesManager;
+use Whitecube\LaravelCookieConsent\Facades\Cookies;
+
 
-class ResetController
-{
-    public function __invoke(Request $request, CookiesManager $cookies)
-    {
-        $response = ! $request->expectsJson()
+class ResetController {
+
+    public function __invoke(Request $request, CookiesManager $cookies) {
+        $response = !$request->expectsJson()
             ? redirect()->back()
             : response()->json([
                 'status' => 'ok',
                 'scripts' => $cookies->getNoticeScripts(true),
                 'notice' => $cookies->getNoticeMarkup(),
             ]);
 
+        $domain = config('cookieconsent.cookie.domain');
```

Comment:
```suggestion
        $consent = config('cookieconsent.cookie.name');
        $domain = config('cookieconsent.cookie.domain');
```

Since we're using `$domain` below, we could also extract the consent cookie name into a variable to make code more readable (see next suggestion).